### DISCUSSION TITLE
part-ii/2020-52/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#329] [Question comments]
Part II 2020 Dynamical Systems: Paper 3, Section II, E 

https://questions.tripos.org/part-ii/2020-52/

**Comment by eha1234** [comment]
For (iii) the rectangle is not in the domain of stability - the contour of V contained in this rectangle is V = 1/2. $\{ V<1/2\}$ is the largest domain we can deduce is contained in the domain of stability of the origin.
